src/utils/share/index.js

```diff
@@ -65,6 +65,10 @@ const copyToClipboard = (username) => {
   Clipboard.setString(`${config.SHARE_URL}/${username}`);
 };
 
+const copyMessageWithoutLink = (message) => {
+  Clipboard.setString(message);
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `copyMessageWithoutLink` function is straightforward and does its job well. However, it's important to consider error handling here. What if the `message` parameter is not a string or is undefined? It would be good to add some basic type checking and error handling to this function.
```diff
- const copyMessageWithoutLink = (message) => {
-   Clipboard.setString(message);
- };
+ const copyMessageWithoutLink = (message) => {
+   if (typeof message !== 'string') {
+     console.error('Invalid argument passed to copyMessageWithoutLink: expected a string');
+     return;
+   }
+   Clipboard.setString(message);
+ };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->